compiler/rustc_middle/src/mir/coverage.rs

```diff
@@ -93,6 +93,10 @@ pub enum CoverageKind {
         op: Op,
         rhs: Operand,
     },
+    Branch {
+        true_op: Operand,
+        false_op: Operand,
+    },
```

Comment:
I think it's a little bit confusing to call these `true_op`/`false_op`, for two reasons:
- `Expression` uses the abbreviation `op` to mean *operation*, not operand
- Even though it makes sense to reuse the `Operand` type here, logically these aren't really “operands”

Perhaps `true_count`/`false_count`?

(In the coverage map parser for #114843 I just punted and called them `true_`/`false_`, which is another option.)

Reply:
Ah, I see you've already used the names `true_counter`/`false_counter` in the data structures that are prepared for FFI. That makes sense, but complicates things.

(Naming this stuff well is hard, especially when LLVM itself isn't very consistent about it!)

Reply:
I renamed half of these to `true_`/`false_`, and I agree that naming these is hard :-)